usaspending_api/download/v2/views.py

```diff
@@ -17,17 +18,41 @@ class RowLimitedIDVDownloadViewSet(BaseDownloadViewSet):
     """
     This route sends a request to the backend to begin generating a zipfile of IDV data in CSV form for download.
     """
+
     endpoint_doc = "usaspending_api/api_docs/api_documentation/download/idv_download.md"
 
     def post(self, request):
         return BaseDownloadViewSet.post(self, request, "idv")
 
 
+class RowLimitedContractDownloadViewSet(BaseDownloadViewSet):
+    """
+    This route sends a request to the backend to begin generating a zipfile of Contract data in CSV form for download.
+    """
+
+    endpoint_doc = "usaspending_api/api_docs/api_documentation/download/contract_download.md"
```

Comment:
It is my understanding that we are moving away from api_documentation and towards having the api_contracts. Would think that you would want to specify `usaspending_api/api_contracts/contracts/v2/download/contract.md` here and `usaspending_api/api_contracts/contracts/v2/download/assistance.md` for the other new endpoint. Would mean that you also don't need to create two markdown files for each new endpoint since the contract will be linked instead of the api_doc.